src/features/SideBarPanel/components/SideBarPanel.jsx

```diff
@@ -0,0 +1,36 @@
+import { Close } from "@carbon/icons-react/next";
+import { Button } from "carbon-components-react";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "../styles/SideBarPanel.scss";
+
+export default function SideBarPanel(props) {
```

Comment:
similarly SideBarPanel is a common component, so it should side inside src/component folder